File: Online_Judge/POJ/2484/Main.java
import java.util.*;

public class Main{
	public static void main(String args[]){
		int n;
		Scanner cin = new Scanner(System.in);
		while(cin.hasNext()){
			n = cin.nextInt();
			if(n == 0)break;
			if(n <= 2)System.out.println("Alice");
			else System.out.println("Bob");
		}
	}
}

File: Online_Judge/POJ/2484/main.cpp
//2017-10-26
#include <cstdio>
#include <cstring>
#include <iostream>
#include <algorithm>

using namespace std;

int main()
{
	int n;
	while(cin>>n && n){
		if(n <= 2)cout<<"Alice"<<endl;
		else cout<<"Bob"<<endl;
	}

	return 0;
}
